src/aap_eda/settings/default.py

```diff
@@ -11,823 +11,39 @@
-            "handlers": ["console"],
-            "level": APP_LOG_LEVEL,
-            "propagate": False,
-        },
-        "ansible_base": {
-            "handlers": ["console"],
-            "level": APP_LOG_LEVEL,
-            "propagate": False,
-        },
-    },
-}
-
-# ---------------------------------------------------------
-# CONTROLLER SETTINGS
-# ---------------------------------------------------------
-
-EDA_CONTROLLER_URL = settings.get("CONTROLLER_URL", "default_controller_url")
-EDA_CONTROLLER_TOKEN = settings.get(
-    "CONTROLLER_TOKEN", "default_controller_token"
-)
-EDA_CONTROLLER_SSL_VERIFY = settings.get("CONTROLLER_SSL_VERIFY", "yes")
-
-# ---------------------------------------------------------
-# RULEBOOK LIVENESS SETTINGS
-# ---------------------------------------------------------
-
-RULEBOOK_READINESS_TIMEOUT_SECONDS = int(
-    settings.get("RULEBOOK_READINESS_TIMEOUT_SECONDS", 60)
-)
-RULEBOOK_LIVENESS_CHECK_SECONDS = int(
-    settings.get("RULEBOOK_LIVENESS_CHECK_SECONDS", 300)
-)
-RULEBOOK_LIVENESS_TIMEOUT_SECONDS = (
-    int(settings.get("RULEBOOK_LIVENESS_TIMEOUT_SECONDS", 310))
-    + RULEBOOK_LIVENESS_CHECK_SECONDS
-)
-ACTIVATION_RESTART_SECONDS_ON_COMPLETE = int(
-    settings.get("ACTIVATION_RESTART_SECONDS_ON_COMPLETE", 0)
-)
-ACTIVATION_RESTART_SECONDS_ON_FAILURE = int(
-    settings.get("ACTIVATION_RESTART_SECONDS_ON_FAILURE", 60)
-)
-ACTIVATION_MAX_RESTARTS_ON_FAILURE = int(
-    settings.get("ACTIVATION_MAX_RESTARTS_ON_FAILURE", 5)
-)
-
-# -1 means no limit
-MAX_RUNNING_ACTIVATIONS = int(settings.get("MAX_RUNNING_ACTIVATIONS", 5))
-
-# ---------------------------------------------------------
-# RULEBOOK ENGINE LOG LEVEL
-# ---------------------------------------------------------
-
-
-# For backwards compatibility, from the old value "-v" to the new value "info"
-def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
-    log_level = settings.get(
-        "ANSIBLE_RULEBOOK_LOG_LEVEL",
-        "error",
-    )
-    if log_level is None:
-        return RulebookProcessLogLevel.ERROR
-    if log_level.lower() == "-v":
-        return RulebookProcessLogLevel.INFO
-    if log_level.lower() == "-vv":
-        return RulebookProcessLogLevel.DEBUG
-    if log_level not in RulebookProcessLogLevel.values():
-        raise ImproperlyConfigured(
-            f"Invalid log level '{log_level}' for ANSIBLE_RULEBOOK_LOG_LEVEL"
-            f" setting. Valid values are: {RulebookProcessLogLevel.values()}"
-        )
-    return RulebookProcessLogLevel(log_level)
+from .post_load import post_loading
 
+EDA_SETTINGS_FILE = "/etc/eda/settings.yaml"
 
-ANSIBLE_RULEBOOK_LOG_LEVEL = get_rulebook_process_log_level()
-ANSIBLE_RULEBOOK_FLUSH_AFTER = settings.get(
-    "ANSIBLE_RULEBOOK_FLUSH_AFTER", 100
+DYNACONF = factory(
+    __name__,
+    "EDA",
+    # Options passed directly to dynaconf
+    environments=("production", "development", "testing"),
+    settings_files=["defaults.py"],
 )
```

Comment:
You can pass `add_dab_settings=False` to skip adding DAB defaults, then later after loading the `constants` you call `load_dab_settings` 

https://github.com/ansible/django-ansible-base/pull/689/files#diff-d7dafe47cdd096243886611525e1f000eca6786be0908ee3c16bbcd08c7bf7a9R25